content/02-understand-prisma/03-prisma-in-your-stack/03-is-prisma-an-orm.mdx

```diff
@@ -1,213 +1,167 @@
 
-| Database     | ORMs                                 | Prisma                                                                                                             |
+In this article, you will learn in more detail about ORM patterns and workflows, how Prisma implements the Data Mapper pattern, and the benefits of Prisma's approach.
+
+<!-- | Database     | Other ORMs               | Prisma                                                                                                             |
 | ------------ | ------------------------------------ | ------------------------------------------------------------------------------------------------------------------ |
-| SQL Schema   | Migrations and model classes         | [Prisma schema](../../reference/tools-and-interfaces/prisma-schema/prisma-schema-file)                             |
+| SQL Schema   | SQL          | [Prisma schema](../../reference/tools-and-interfaces/prisma-schema/prisma-schema-file)                             |
 | Tables       | Model classes                        | [Models](../../reference/tools-and-interfaces/prisma-schema/data-model#defining-models) in the Prisma schema       |
 | Columns      | Properties in model classes          | Model [fields](../../reference/tools-and-interfaces/prisma-schema/data-model#defining-fields) in the Prisma schema |
 | Records      | Instances of a model class (objects) | Plain JavaScript objects                                                                                           |
-| Foreign keys | Foreign keys on model classes        | [Relations](../../reference/tools-and-interfaces/prisma-schema/relations) between Prisma models                    |
-
-In this article, you will learn in more detail about the differences of Prisma compared to traditional ORMs and other database access libraries.
+| Foreign keys | Foreign keys on model classes        | [Relations](../../reference/tools-and-interfaces/prisma-schema/relations) between Prisma models                    | -->
 
 ## What are ORMs?
 
 If you're already familiar with ORMs, feel free to jump to the [next section](#prisma) on Prisma.
 
-### Concepts
+### ORM Patterns - Active Record and Data Mapper
+
+ORMs provide a high-level database abstraction. They expose a programmatic interface through objects to create, read, delete, and manipulate data while hiding some of the complexity of the database.
+
+The idea with ORMs is that you define your models as **classes** that map to tables in a database. The classes and their instances provide you with a flexible API to read and write data in the database. This is necessary because objects and relational databases have different mechanisms for structuring data.
```

Comment:
```suggestion
The idea with ORMs is that you define your models as **classes** that map to tables in a database. The classes and their instances provide you with a programmatic API to read and write data in the database. This is necessary because objects and relational databases have different mechanisms for structuring data.
```